src/uproot_browser/plot.py

```diff
@@ -57,6 +57,7 @@ def plot_branch(tree: uproot.TBranch) -> None:
     finite = values[np.isfinite(values)]
     histogram: hist.Hist = hist.numpy.histogram(finite, bins=100, histogram=hist.Hist)
     plt.bar(histogram.axes[0].centers, histogram.values().astype(float))
+    plt.ylim(0, np.max(histogram.values()))
```

Comment:
I think you can just use 
```suggestion
    plt.ylim(lower = 0)
```

Which saves the cost of computing the max. (I think this is called `bottom` in mpl, actually? If there's truly a mismatch then it should either be fixed in plotext or we need to support both, since `--iterm` will use mpl instead).

https://github.com/piccolomo/plotext/blob/d3f181311dcc4abe66662d70dc833fc471fef05f/plotext/_figure.py#L124

And 

https://github.com/matplotlib/matplotlib/blob/4f5cacfde9024076d9ab0cb6ad1c9a7cf352d5f9/lib/matplotlib/pyplot.py#L1724-L1758 which calls https://github.com/matplotlib/matplotlib/blob/4f5cacfde9024076d9ab0cb6ad1c9a7cf352d5f9/lib/matplotlib/axes/_base.py#L3948

Actually, are xlim and ylim's names reversed in plotext too?

Reply:
I think they might be reversed, yes. `plt.ylim(lower = 0)` does not work, [`ylim`](https://github.com/piccolomo/plotext/blob/d3f181311dcc4abe66662d70dc833fc471fef05f/plotext/_figure.py#L127) expects `left` and `right` rather.

Reply:
Would a PR to plotext, both fixing the names for x and y and using the mpl name for y be in order?

Reply:
That makes sense, I'll open one.